.github/workflows/infrastructure.yml

```diff
@@ -0,0 +1,171 @@
+      if: contains('["dev", "staging"]', env.target)
+      run: echo "type=Preproduction" >> $GITHUB_ENV
+
+    - name: Production 💫
+      if: ${{ 'prod' == env.target }}
+      run: echo "type=Production" >> $GITHUB_ENV
+
+    - name: Tags 🏷️
+      run: echo tags='Environment=${{ env.type }}' \
+        'Product=${{ env.product }}' \
+        'Team=development' >> $GITHUB_ENV
+
+    - name: Login 🔐
+      uses: azure/login@v1
+      with:
+        creds: ${{ secrets.AZURE_CREDENTIALS }}
+
+    - name: Azure defaults ✨
+      uses: Azure/cli@1.0.4
+      with:
+        inlineScript: |
+          # Basic
+          az configure --defaults location=${{ vars.REGION }}
+          az configure --defaults group=rg-${{ env.product }}-${{ env.target }}-001
+
+          # AZ CLI extensions upgrade
+          az extension add --name containerapp --upgrade
+
+    - name: Resource group 🏗️
+      uses: Azure/cli@1.0.4
+      with:
+        inlineScript: |
+          az group create \
+          --name rg-${{ env.product }}-${{ env.target }}-001 \
+          --tags ${{ env.tags }}
+
+    
+    - name: Container registry 📦️
+      uses: Azure/cli@1.0.4
+      with:
+        inlineScript: |
+          az acr create \
+          --name cr${{ env.product }}${{ env.target }}001 \
+          --sku Standard \
+          --admin-enabled true \
+          --tags ${{ env.tags }}
+
+    - name: Virtual network 🧵
+      uses: Azure/cli@1.0.4
+      with:
+        inlineScript: |
+          az network vnet create \
+          --name vnet-${{ env.product }}-001 \
+          --address-prefix ${{ secrets.VNET_ADDRESS_PREFIX }} \
+          --subnet-name snet-${{ env.product }}-001 \
+          --subnet-prefixes ${{ secrets.VNET_SUBNET_PREFIX }} \
+          --tags ${{ env.tags }}
+          
+    # - name: VNET peer 🔀
+    #   uses: Azure/cli@1.0.4
+    #   with:
+    #     inlineScript: |
+    #       az network vnet peering create \
+    #       --name vnet-peer-${{ env.product }}-001 \
+    #       --vnet-name vnet-${{ env.product }}-001 \
+    #       --remote-vnet $(az network vnet show --subscription ${{ secrets.REMOTE_VNET_SUBSCRIPTION }} --resource-group ${{ secrets.VNET_REMOTE_RESOURCE_GROUP }} --name ${{ secrets.REMOTE_VNET }} --query 'id' -o tsv) \
+    #       --allow-forwarded-traffic 1 \
+    #       --allow-vnet-access 1
+
+    - name: Container app environment 🗃️
+      uses: Azure/cli@1.0.4
+      with:
+        inlineScript: |
+          az containerapp env create \
+          --name cae-${{ env.product }}-${{ env.target }}-001 \
+          --infrastructure-subnet-resource-id $(az network vnet subnet show --name snet-${{ env.product }}-001 --vnet-name vnet-${{ env.product }}-001 --query 'id' -o tsv) \
+          --tags ${{ env.tags }}
+
+    
+    - name: Container app - MDM 📄
+      uses: Azure/cli@1.0.4
+      with:
+        inlineScript: |
+          az containerapp create \
+          --name ca-${{ env.product }}-mdm-${{ env.target }}-001 \
```

Comment:
@IgnatG left a [comment](https://github.com/UK-Export-Finance/mdm-api/pull/58/files/d2f7b41f79ad4e7f18174f3a508e1893f2c17257#r1143513462) on a previous version of this code saying that we should remove `mdm` because `env.product` should be enough - have we decided to keep `mdm` now?